alphadia/workflow/peptidecentric.py

```diff
@@ -894,6 +897,8 @@ def extraction(self):
 
         self.log_precursor_df(precursor_df)
 
+        self.timing_manager.end("extraction")
```

Comment:
if `self.timing_manager.start("extraction")` was called again, this would overwrite the previously recorded timeing, right? is this intended? or should we change it such that it accumulates?

Reply:
I suppose it would. Why would we want to start the same timing twice though (unless we were trying to time iterations of a loop or something like that, but then I think we would need to make the whole logic of the timing manager considerably more complex)?